frontend/src/metabase/selectors/settings.ts

```diff
@@ -26,6 +31,54 @@ export const getLearnUrl = (path = "") => {
   return `https://www.metabase.com/learn/${path}`;
 };
 
+interface DocsUrlProps {
+  page?: string;
+  anchor?: string;
+}
+
+export const getDocsUrl = createSelector(
```

Comment:
Moved from `MetabaseSettings`. Added `getDocsUrlForVersion` to avoid code duplication between the new and deprecated implementation